src/stores/factories/queries/createFindSafesByOwner.ts

```diff
@@ -49,20 +49,14 @@ async function queryCirclesGarden(ownerAddress: string, safeAddresses: string[])
 }
 
 async function queryCirclesSubgraph(ownerAddress: string) {
-    const args = {
-        "headers": {
-            "Accept": "application/json",
-            "Accept-Language": "en-US,en;q=0.5",
-            "content-type": "application/json",
-        },
-        "body": "{\"query\":\"{\\n  user(id: \\\"" + ownerAddress.toLowerCase() + "\\\") {\\n    safeAddresses\\n  }\\n}\",\"variables\":null,\"extensions\":{\"headers\":null}}",
-        "method": "POST"
-    };
-
-    const safesByOwnerResult = await fetch(CirclesSubgraphApi, args);
+    const web3 = new Web3();
+    const checksumAddress = web3.utils.toChecksumAddress(ownerAddress);
+    const requestUrl = `https://safe-transaction-gnosis-chain.safe.global/api/v1/owners/${checksumAddress}/safes/`;
```

Comment:
Can you put the url into the consts file (can also be a function that builds this URL using the checksumAddress). It's just better to have such values in one place.